### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by CusiniM**
Let's try to have as much granularity as possible and multiple people per folder. I have also started to remove myself and randy from a few of these so that we don't get asked to review basically every PR.  

I have broken down `inputFiles` and  `constitutive`. 

2 things to keep in mind:
 - deepest level rules
 - if we leave a folder blank it means no one owns it. I think this might be a good solution for some docs folders.
 

Go ahead and add yourselves wherever you see fit.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -10,8 +10,37 @@
...
+/inputFiles/compositionalMultiphaseFlow
+/inputFiles/compositionalMultiphaseWell
+/inputFiles/efemFractureMechanics                    @CusiniM
+/inputFiles/hydraulicFracturing                      @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman
+/inputFiles/inducedSeismicity                        @CusiniM @matteofrigo5 
+/inputFiles/initialization
+/inputFiles/lagrangianContactMechanics               @CusiniM
+/inputFiles/materialPointMethod
+/inputFiles/meshGeneration
+/inputFiles/multiphaseFlowFractures
```
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/multiphaseFlowFractures               @Cusinim @paveltomin 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -11,7 +11,36 @@
...
 /.integrated_tests.yaml                               @CusiniM @cssherman @rrsettgast @wrtobin @castelletto1 @jhuang2601 @paveltomin
-/inputFiles/                                          @CusiniM @cssherman @jhuang2601 @rrsettgast
+/inputFiles/compositionalMultiphaseFlow               @paveltomin
+/inputFiles/compositionalMultiphaseWell               @paveltomin
+/inputFiles/efemFractureMechanics                     @CusiniM
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization
+/inputFiles/lagrangianContactMechanics                @CusiniM
+/inputFiles/materialPointMethod
```
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/materialPointMethod                @homel1 @cmcrook5 
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -11,7 +11,36 @@
 
 /BASELINE_NOTES.md
 /.integrated_tests.yaml                               @CusiniM @cssherman @rrsettgast @wrtobin @castelletto1 @jhuang2601 @paveltomin
-/inputFiles/                                          @CusiniM @cssherman @jhuang2601 @rrsettgast
+/inputFiles/compositionalMultiphaseFlow               @paveltomin
+/inputFiles/compositionalMultiphaseWell               @paveltomin
+/inputFiles/efemFractureMechanics                     @CusiniM
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman @jhuang2601
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/initialization                                 @jhuang2601 
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/lagrangianContactMechanics                @CusiniM
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601 
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/materialPointMethod
+/inputFiles/meshGeneration
+/inputFiles/multiphaseFlowFractures                   @paveltomin
+/inputFiles/multipleMeshBodies
+/inputFiles/phaseField                                @frankfeifan @CusiniM
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601
```

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin @jhuang2601
```

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/compositionalMultiphaseFlow               @paveltomin
+/inputFiles/compositionalMultiphaseWell               @paveltomin
+/inputFiles/efemFractureMechanics                     @CusiniM
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization
+/inputFiles/lagrangianContactMechanics                @CusiniM
+/inputFiles/materialPointMethod
+/inputFiles/meshGeneration
+/inputFiles/multiphaseFlowFractures                   @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/multiphaseFlowFractures                   @paveltomin @jhuang2601
```

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/multipleMeshBodies
+/inputFiles/phaseField                                @frankfeifan @CusiniM
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin
+/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin
+/inputFiles/proppant
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/proppant                                  @jhuang2601 
```

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/relpermDriver
+/inputFiles/simplePDE
+/inputFiles/singlePhaseFlow                           @CusiniM @castelletto1 @paveltomin
+/inputFiles/singlePhaseFlowFractures                  @CusiniM @castelletto1 @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/singlePhaseFlowFractures                  @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/singlePhaseWell                           @paveltomin
+/inputFiles/solidMechanics                            @CusiniM @castelletto1
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/solidMechanics                            @CusiniM @castelletto1 @jhuang2601 
```

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/surfaceGeneration                         @CusiniM @castelletto1
+/inputFiles/thermalMultiphaseFlow                     @paveltomin
+/inputFiles/thermalSinglePhaseFlowFractures           @CusiniM @castelletto1 @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermalSinglePhaseFlowFractures           @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/thermoPoromechanics                       @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermoPoromechanics                       @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/thermoPoromechanicsFractures              @CusiniM @castelletto1 @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/thermoPoromechanicsFractures              @CusiniM @castelletto1 @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS [hunk 15]
```diff
@@ -11,7 +11,36 @@
...
+/inputFiles/triaxialDriver
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/triaxialDriver                 @jhuang2601
```

File: .github/CODEOWNERS [hunk 16]
```diff
@@ -32,10 +61,10 @@
...
 /src/coreComponents/physicsSolvers/contact            @CusiniM @rrsettgast @matteofrigo5
 /src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @francoishamon
-/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast
-/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast @matteofrigo5
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan
 /src/coreComponents/physicsSolvers/python             @cssherman @corbett5
-/src/coreComponents/physicsSolvers/simplePDE          @castelletto1 @rrsettgast @CusiniM 
+/src/coreComponents/physicsSolvers/simplePDE          @castelletto1 @rrsettgast @CusiniM @frankfeifan
 /src/coreComponents/physicsSolvers/solidMechanics     @rrsettgast @castelletto1 @CusiniM
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/solidMechanics     @rrsettgast @castelletto1 @CusiniM @jhuang2601
```

File: .github/CODEOWNERS [hunk 17]
```diff
@@ -32,10 +61,10 @@
 /src/coreComponents/physicsSolvers                    @rrsettgast @CusiniM
 /src/coreComponents/physicsSolvers/contact            @CusiniM @rrsettgast @matteofrigo5
 /src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @francoishamon
-/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast
-/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast @matteofrigo5
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan @jhuang2601
```

**Review by jhuang2601 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 18]
```diff
@@ -11,11 +11,52 @@
...
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman @jhuang2601
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization                            @jhuang2601 
+/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601 
+/inputFiles/materialPointMethod                       @homel1 @cmcrook5 
+/inputFiles/meshGeneration
+/inputFiles/multiphaseFlowFractures                   @paveltomin @jhuang2601
+/inputFiles/multipleMeshBodies
+/inputFiles/phaseField                                @frankfeifan @CusiniM
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601
```
Comment (ryar9534) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601 @ryar9534 
```

File: .github/CODEOWNERS [hunk 19]
```diff
@@ -32,10 +73,10 @@
 /src/coreComponents/physicsSolvers                    @rrsettgast @CusiniM
 /src/coreComponents/physicsSolvers/contact            @CusiniM @rrsettgast @matteofrigo5
 /src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @francoishamon
-/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast
-/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast @matteofrigo5
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan
```
Comment (ryar9534) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan @ryar9534 
```

**Review by ryar9534 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 20]
```diff
@@ -11,31 +11,72 @@
 
 /BASELINE_NOTES.md
 /.integrated_tests.yaml                               @CusiniM @cssherman @rrsettgast @wrtobin @castelletto1 @jhuang2601 @paveltomin
-/inputFiles/                                          @CusiniM @cssherman @jhuang2601 @rrsettgast
```
Comment (rrsettgast) [inline .github/CODEOWNERS:14]: you should have a default for the subdirectories that don't have a listing

File: .github/CODEOWNERS [hunk 21]
```diff
@@ -11,31 +11,72 @@
...
+/inputFiles/compositionalMultiphaseFlow               @paveltomin
+/inputFiles/compositionalMultiphaseWell               @paveltomin
+/inputFiles/efemFractureMechanics                     @CusiniM
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman @jhuang2601
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization                            @jhuang2601
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/initialization                            @jhuang2601 @rrsettgast
```

File: .github/CODEOWNERS [hunk 22]
```diff
@@ -11,31 +11,72 @@
...
 /BASELINE_NOTES.md
 /.integrated_tests.yaml                               @CusiniM @cssherman @rrsettgast @wrtobin @castelletto1 @jhuang2601 @paveltomin
-/inputFiles/                                          @CusiniM @cssherman @jhuang2601 @rrsettgast
+/inputFiles/compositionalMultiphaseFlow               @paveltomin
+/inputFiles/compositionalMultiphaseWell               @paveltomin
+/inputFiles/efemFractureMechanics                     @CusiniM
+/inputFiles/hydraulicFracturing                       @CusiniM @rrsettgast @frankfeifan @Guotong-Ren @cssherman @jhuang2601
+/inputFiles/inducedSeismicity                         @CusiniM @matteofrigo5 
+/inputFiles/initialization                            @jhuang2601 
+/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/lagrangianContactMechanics                @CusiniM @jhuang2601 @matteofrigo5 
```

File: .github/CODEOWNERS [hunk 23]
```diff
@@ -11,31 +11,72 @@
...
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601 @ryar9534 
+/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin @jhuang2601
+/inputFiles/proppant                                  @jhuang2601 
+/inputFiles/relpermDriver
+/inputFiles/simplePDE
+/inputFiles/singlePhaseFlow                           @CusiniM @castelletto1 @paveltomin
+/inputFiles/singlePhaseFlowFractures                  @CusiniM @castelletto1 @paveltomin @jhuang2601 
+/inputFiles/singlePhaseWell                           @paveltomin
+/inputFiles/solidMechanics                            @CusiniM @castelletto1 @jhuang2601 
+/inputFiles/surfaceGeneration                         @CusiniM @castelletto1
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/surfaceGeneration                         @CusiniM @castelletto1 @rrsettgast 
```

File: .github/CODEOWNERS [hunk 24]
```diff
@@ -11,31 +11,72 @@
...
-/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @joshua-white
+/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @castelletto1
+/src/coreComponents/constitutive/capillaryPressure    @CusiniM @dkachuma @paveltomin @joshua-white
+/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
+/src/coreComponents/constitutive/diffusion            @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/dispersion           @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/docs                  
+/src/coreComponents/constitutive/fluid                @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/permeability         @rrsettgast @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/pvtPackage           @rrsettgast @CusiniM @dkachuma @paveltomin @joshua-white
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin 
/src/coreComponents/constitutive/pvtPackage           @CusiniM @dkachuma @paveltomin @joshua-white
```

File: .github/CODEOWNERS [hunk 25]
```diff
@@ -11,31 +11,72 @@
...
 /src/coreComponents/mesh                              @rrsettgast @wrtobin @untereiner @CusiniM 
 /src/coreComponents/physicsSolvers                    @rrsettgast @CusiniM
 /src/coreComponents/physicsSolvers/contact            @CusiniM @rrsettgast @matteofrigo5
-/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @francoishamon
-/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast
-/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast
+/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @ryar9534
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast @matteofrigo5
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast @frankfeifan @ryar9534 
 /src/coreComponents/physicsSolvers/python             @cssherman @corbett5
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/python             @cssherman @corbett5 @alexbenedicto 
```

File: .github/CODEOWNERS [hunk 26]
```diff
@@ -11,31 +11,72 @@
...
+/src/coreComponents/constitutive/relativePermeability @dkachuma @paveltomin 
+/src/coreComponents/constitutive/solid                @rrsettgast @CusiniM @paveltomin @joshua-white @castelletto1
+/src/coreComponents/constitutive/thermalConductivity  @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/unitTests            @rrsettgast @CusiniM @dkachuma @paveltomin 
 /src/coreComponents/dataRepository                    @rrsettgast @wrtobin @corbett5
 /src/coreComponents/denseLinearAlgebra                @rrsettgast @CusiniM @castelletto1 
 /src/coreComponents/discretizationMethods             @rrsettgast @CusiniM @castelletto1 
 /src/coreComponents/events                            @rrsettgast @corbett5 @cssherman 
 /src/coreComponents/fieldSpecification                @rrsettgast @CusiniM @corbett5 @cssherman 
 /src/coreComponents/fileIO                            @rrsettgast @wrtobin @MelReyCG @untereiner
```
Comment (rrsettgast) [inline .github/CODEOWNERS:67]: Who is in charge of the vtk interface?

File: .github/CODEOWNERS [hunk 27]
```diff
@@ -11,31 +11,72 @@
...
 /src/coreComponents/finiteElement                     @rrsettgast @castelletto1 @andrea-borio @CusiniM
-/src/coreComponents/finiteVolume                      @rrsettgast @castelletto1 @CusiniM @paveltomin
+/src/coreComponents/finiteVolume                      @rrsettgast @castelletto1 @CusiniM @paveltomin @ryar9534
 /src/coreComponents/functions                         @cssherman  @wrtobin @rrsettgast
 /src/coreComponents/linearAlgebra                     @rrsettgast @castelletto1 @victorapm
```
Comment (rrsettgast) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/linearAlgebra                     @rrsettgast @castelletto1 @victorapm  @CusiniM 
```

**Review by rrsettgast — APPROVED** [review]
The problem with having a bunch of different folks as code owners for the various inputFiles subdirectories is that a sweeping change will be very hard to get approved.

@CusiniM Perhaps you and I should be on every code owner spec? I generally wait until others have done their reviews, so this doesn't have a "randy approved so it is good" effect.

File: .github/CODEOWNERS (hunk 20 above)
Comment (CusiniM) [reply .github/CODEOWNERS:14]: this change was not intended.

**Comment by CusiniM** [comment]
> The problem with having a bunch of different folks as code owners for the various inputFiles subdirectories is that a sweeping change will be very hard to get approved.
> 

Yeah, I had not considered this. On the other hand, if a change affects all inputFiles maybe it's good to require a lot of codeowners coz it ensures that everyone's aware of it? 

> @CusiniM Perhaps you and I should be on every code owner spec? I generally wait until others have done their reviews, so this doesn't have a "randy approved so it is good" effect.

I was trying to reduce the amount of notifications we receive so that it's easier to identify what really needs attention. Maybe one of us is sufficient?

File: .github/CODEOWNERS (hunk 20 above)
Comment (CusiniM) [reply .github/CODEOWNERS:14]: fixed.

File: .github/CODEOWNERS (hunk 26 above)
Comment (CusiniM) [reply .github/CODEOWNERS:67]: I am not sure.

File: .github/CODEOWNERS [hunk 28]
```diff
@@ -10,37 +10,79 @@
...
+/inputFiles/materialPointMethod                       @homel1 @cmcrook5 
+/inputFiles/meshGeneration                            @rrsettgast @cssherman
+/inputFiles/multiphaseFlowFractures                   @CusiniM @paveltomin @jhuang2601
+/inputFiles/multipleMeshBodies                        @rrsettgast @wrtobin
+/inputFiles/phaseField                                @CusiniM @frankfeifan 
+/inputFiles/poromechanics                             @CusiniM @castelletto1 @paveltomin @jhuang2601 @ryar9534 
+/inputFiles/poromechanicsFractures                    @CusiniM @castelletto1 @paveltomin @jhuang2601
+/inputFiles/proppant                                  @rrsettgast @jhuang2601 
+/inputFiles/relpermDriver                             @dkachuma @paveltomin @jafranc
+/inputFiles/simplePDE
```
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/inputFiles/simplePDE                                 @rrsettgast @castelletto1 
```

File: .github/CODEOWNERS [hunk 29]
```diff
@@ -10,37 +10,79 @@
...
-/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @klevzoff @francoishamon
-/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM @rrsettgast
-/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 @rrsettgast
-/src/coreComponents/physicsSolvers/python             @cssherman @corbett5
-/src/coreComponents/physicsSolvers/simplePDE          @castelletto1 @rrsettgast @CusiniM 
-/src/coreComponents/physicsSolvers/solidMechanics     @rrsettgast @castelletto1 @CusiniM 
+/src/coreComponents/physicsSolvers/contact            @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
+/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin @ryar9534
+/src/coreComponents/physicsSolvers/inducedSeismicity  @rrsettgast @CusiniM @matteofrigo5 @VidarStiernstrom
+/src/coreComponents/physicsSolvers/multiphysics       @rrsettgast @CusiniM @paveltomin @castelletto1 @frankfeifan @ryar9534
```
Comment (CusiniM) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/physicsSolvers/multiphysics       @rrsettgast @CusiniM @paveltomin @castelletto1 @frankfeifan @ryar9534 @jhuang2601
```

File: .github/CODEOWNERS [hunk 30]
```diff
@@ -10,37 +10,79 @@
...
+/inputFiles/wavePropagation                           @acitrain @sframba
+/inputFiles/wellbore                                  @rrsettgast @jhuang2601
+/inputFiles/wellboreECP                               @rrsettgast 
+/src/coreComponents/LvArray                           @rrsettgast @CusiniM @wrtobin @corbett5 
 /src/coreComponents/codingUtilities                   @rrsettgast @untereiner @corbett5 @wrtobin
 /src/coreComponents/common                            @rrsettgast @MelReyCG @corbett5 @wrtobin
-/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @joshua-white
+/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @castelletto1
+/src/coreComponents/constitutive/capillaryPressure    @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren @jhuang2601 
```

File: .github/CODEOWNERS [hunk 31]
```diff
@@ -10,37 +10,79 @@
...
+/src/coreComponents/constitutive/diffusion            @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/dispersion           @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/docs
```
Comment (jhuang2601) [inline .github/CODEOWNERS:53]:
```suggestion
/src/coreComponents/constitutive/docs                  @jhuang2601 
```

File: .github/CODEOWNERS [hunk 32]
```diff
@@ -10,37 +10,79 @@
...
 /src/coreComponents/common                            @rrsettgast @MelReyCG @corbett5 @wrtobin
-/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @joshua-white
+/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma @paveltomin @castelletto1
+/src/coreComponents/constitutive/capillaryPressure    @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
+/src/coreComponents/constitutive/diffusion            @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/dispersion           @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/docs                  
+/src/coreComponents/constitutive/fluid                @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin @jhuang2601 
```

File: .github/CODEOWNERS [hunk 33]
```diff
@@ -10,37 +10,79 @@
...
+/src/coreComponents/constitutive/capillaryPressure    @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/contact              @rrsettgast @CusiniM @matteofrigo5 @Guotong-Ren
+/src/coreComponents/constitutive/diffusion            @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/dispersion           @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/docs                  
+/src/coreComponents/constitutive/fluid                @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/pvtPackage           @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/relativePermeability @dkachuma @paveltomin 
+/src/coreComponents/constitutive/solid                @rrsettgast @CusiniM @paveltomin @castelletto1
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/solid                @rrsettgast @CusiniM @paveltomin @castelletto1 @jhuang2601 
```

File: .github/CODEOWNERS [hunk 34]
```diff
@@ -10,37 +10,79 @@
...
+/src/coreComponents/constitutive/thermalConductivity  @CusiniM @dkachuma @paveltomin
```
Comment (jhuang2601) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/constitutive/thermalConductivity  @CusiniM @dkachuma @paveltomin @jhuang2601 
```

**Review by jhuang2601 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 31 above)
Comment (CusiniM) [reply .github/CODEOWNERS:53]: I would leave this blank so that it's excluded from the codeowners lists.

**Review by dkachuma — COMMENTED** [review]
Can we also add owners to `src/coreComponents/constitutiveDrivers`

**Comment by CusiniM** [comment]
> Can we also add owners to `src/coreComponents/constitutiveDrivers`

yes, of course.

File: .github/CODEOWNERS [hunk 35]
```diff
@@ -10,37 +10,80 @@
...
+/inputFiles/thermalMultiphaseFlow                     @paveltomin @dkachuma
+/inputFiles/thermalSinglePhaseFlowFractures           @CusiniM @castelletto1 @paveltomin @jhuang2601 @frankfeifan
+/inputFiles/thermoPoromechanics                       @paveltomin @jhuang2601 @castelletto1 @frankfeifan
+/inputFiles/thermoPoromechanicsFractures              @CusiniM @castelletto1 @paveltomin @jhuang2601 
+/inputFiles/triaxialDriver                            @rrsettgast @jhuang2601
+/inputFiles/wavePropagation                           @acitrain @sframba
+/inputFiles/wellbore                                  @rrsettgast @jhuang2601
+/inputFiles/wellboreECP                               @rrsettgast 
+/src/coreComponents/LvArray                           @rrsettgast @CusiniM @wrtobin @corbett5 
 /src/coreComponents/codingUtilities                   @rrsettgast @untereiner @corbett5 @wrtobin
```
Comment (MelReyCG) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/codingUtilities                   @rrsettgast @untereiner @corbett5 @wrtobin @MelReyCG
```

File: .github/CODEOWNERS [hunk 36]
```diff
@@ -10,37 +10,80 @@
...
+/src/coreComponents/constitutive/docs                  
+/src/coreComponents/constitutive/fluid                @CusiniM @dkachuma @paveltomin 
+/src/coreComponents/constitutive/permeability         @CusiniM @dkachuma @paveltomin @jhuang2601 
+/src/coreComponents/constitutive/pvtPackage           @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutive/relativePermeability @dkachuma @paveltomin 
+/src/coreComponents/constitutive/solid                @rrsettgast @CusiniM @paveltomin @castelletto1 @jhuang2601 
+/src/coreComponents/constitutive/thermalConductivity  @CusiniM @dkachuma @paveltomin @jhuang2601 
+/src/coreComponents/constitutive/unitTests            @rrsettgast @CusiniM @dkachuma @paveltomin
+/src/coreComponents/constitutiveDrivers               @rrsettgast @CusiniM @dkachuma @jhuang2601
 /src/coreComponents/dataRepository                    @rrsettgast @wrtobin @corbett5
```
Comment (MelReyCG) [inline .github/CODEOWNERS]:
```suggestion
/src/coreComponents/dataRepository                    @rrsettgast @wrtobin @corbett5 @MelReyCG 
```

**Review by MelReyCG — COMMENTED** [review]
After discussing with @rrsettgast and @herve-gross, I would like to propose myself on the following folders:
```
/src/coreComponents/codingUtilities    
/src/coreComponents/dataRepository
```

**Review by rrsettgast — APPROVED** [review]

**Pull request merged**